src/api/app/views/webui2/shared/_sign_up_form_input.html.haml

```diff
@@ -0,0 +1,14 @@
+.form-group
```

Comment:
We need to let our users know which fields are required. For an example to see how we did it before, please have a look at #7017.